components/form-components/ScrollView.tsx

```diff
@@ -17,6 +17,7 @@ export const TiFFormScrollView = ({
   <KeyboardAwareScrollView
     style={[style, styles.scroll]}
     contentContainerStyle={styles.contentContainer}
+    enableResetScrollToCoords={false}
```

Comment:
`enableResetScrollToCoords` sounds a little vague; does this mean that if the keyboard is used on the map component, it'd reset the coordinates after use of the keyboard?

Reply:
This fixes a problem with the edit event form, in which if you dismissed the keyboard, it would scroll to the top of the screen for some reason. Obviously, that was very annoying.